tests/02_generic_input/10_include_csv.yaml

```diff
@@ -54,7 +54,7 @@ problem: !Coupling_Sequential
                 format:
                   t:
                     time: $0
-                  value: $1
```

Comment:
Add note , that CSV reader is not compatible with automatic conversion to the tensor, which is the input shape of all FieldConstant and FieldTimeFunction fields.